CudaArchitectureSelector.cmake

```diff
@@ -230,14 +238,15 @@ function(cas_get_architectures_by_name name output)
     set( kepler_version 3)
     set(maxwell_version 5)
     set( pascal_version 6)
-    set(  volta_version 7)
+    set(  volta_version "7(0|2)")
```

Comment:
extra space

```suggestion
    set( volta_version "7(0|2)")
```

Reply:
These were done on purpose to align all the versions